packages/core/src/node/initRsbuild.ts

```diff
@@ -160,15 +160,21 @@ async function createInternalBuildConfig(
     },
     tools: {
       bundlerChain(chain, { CHAIN_ID }) {
-        const swcLoaderOptions = chain.module
-          .rule(CHAIN_ID.RULE.JS)
+        const jsModuleRule = chain.module.rule(CHAIN_ID.RULE.JS);
+
+        const swcLoaderOptions = jsModuleRule
           .use(CHAIN_ID.USE.SWC)
           .get('options');
 
         chain.module
           .rule('MDX')
           .type('javascript/auto')
           .test(MDX_REGEXP)
+          .resolve.merge({
```

Comment:
can we just inherit the `resolve` config

Reply:
can't get resolve config by `rule(js).get("resolve")`
<img width="811" alt="image" src="https://github.com/web-infra-dev/rspress/assets/22373761/daf2c2ac-421f-48c3-b93b-019365d72d55">

Reply:
ok